openapi-cli/src/main/java/org/ballerinalang/openapi/cmd/OpenApiCmd.java

```diff
@@ -92,33 +93,43 @@
     public OpenApiCmd() {
         this.outStream = System.err;
         this.executionPath = Paths.get(System.getProperty("user.dir"));
+        this.exitWhenFinish = true;
     }
 
     public OpenApiCmd(PrintStream outStream, Path executionDir) {
         this.outStream = outStream;
         this.executionPath = executionDir;
+        this.exitWhenFinish = true;
```

Comment:
These 2 constructors can be changed to call the `public OpenApiCmd(PrintStream outStream, Path executionDir, boolean exitWhenFinish)` constructor rather than assigning to each class field.